MIGRATION.md

```diff
@@ -0,0 +1,222 @@
+# Migration Guide for Azure Data Explorer (Kusto) Go SDK
+
+Welcome to the migration guide for the Azure Data Explorer (Kusto) Go SDK. This guide is designed to assist you in migrating your application from using the beta version of the SDK (`github.com/Azure/azure-kusto-go`) to the new, stable release, version `1.0.0`.
+
+The release of version `1.0.0` introduces breaking changes, which include changes in dependencies, package arrangement, querying methods, and management commands. Following these steps carefully will ensure a smooth transition to the new version.
+
+## 1. Changes in Dependencies and go.mod File
+
+To migrate from the old SDK to the new one, you'll need to update your `go.mod` file's dependencies.
+
+**Old SDK:**
+```bash
+go get github.com/Azure/azure-kusto-go
+```
+
+**New SDK:**
+- For data operations (`github.com/Azure/azure-kusto-go/azkustodata`):
+```bash
+go get github.com/Azure/azure-kusto-go/azkustodata
+```
+
+- For ingestion operations (`github.com/Azure/azure-kusto-go/azkustoingest`):
+```bash
+go get github.com/Azure/azure-kusto-go/azkustoingest
+```
+
+Manually update your go.mod file by replacing `github.com/Azure/azure-kusto-go` with the specific package(s) you need.
+
+## 2. Changes in Package Arrangement
+
+The SDK is now split into two separate packages:
+- **azkustodata:** For querying and managing Azure Data Explorer clusters.
+- **azkustoingest:** For ingesting data into Azure Data Explorer clusters.
+
+### Importing the New Packages
+
+Depending on your requirements, import one or both of these packages into your Go files.
+
+**For Data Operations:**
+```go
+import (
+    "github.com/Azure/azure-kusto-go/azkustodata"
+)
+```
+
+**For Ingestion Operations:**
+```go
+import (
+    "github.com/Azure/azure-kusto-go/azkustoingest"
+)
+```
+
+### Using the New Packages
+
+Update your code to use the new packages.
```

Comment:
```suggestion
Update your code with the new package API
```

Reply:
I don't think this sentence is better, the point is the packages themselves are new